compiler/pipes/calc-bad-vars.cpp

```diff
@@ -684,8 +696,12 @@ class CalcBadVars {
 
     {
       FuncCallGraph call_graph(std::move(functions), dep_datas);
-      calc_interruptible(call_graph);
-      calc_resumable(call_graph, dep_datas);
+      if (G->is_output_mode_k2_component()) {
+        calc_interruptible(call_graph);
```

Comment:
Let's change order of calc calls. Because code like this doesn't compile since `kek` doesn't mark as interruptible
```php
<?php

function my_fork($level) {
    return true;
}

function kek() {
    return my_fork(0);
}

$head = fork(my_fork(0));
wait($head);

kek();
```